numpy/array_api/_array_object.py

```diff
@@ -979,6 +995,11 @@ def __rxor__(self: Array, other: Union[int, bool, Array], /) -> Array:
         res = self._array.__rxor__(other._array)
         return self.__class__._new(res)
 
+    def to_device(self: Array, device: Device, /) -> Array:
```

Comment:
Not critical for this PR, but `Device` is currently defined as an alias for `Any`. Based on how it's used throughout the API implementation you should be able to narrow it down to a string-literal.

``` diff
- Device = Any
+ Device = typing.Literal["cpu"]
```

Reply:
I've done this. It's Any in the general array API but for NumPy it is just "cpu" (the device functionality does basically nothing in the NumPy implementation).